src/Components/PresentationalComponents/CSAwRuleBox/CSAwRuleBox.js

```diff
@@ -72,9 +72,10 @@ const CSAwRuleBox = ({ rules, synopsis, changeExposedSystemsParameters, intl })
                                             }}
                                             component={TextVariants.small}
                                         >
-                                            <Link
+                                            <HashLink
```

Comment:
I believe this can be simply rewritten to
```
<a href="#systems-exposed-table-header> ... </a>
```
This way, `a` mustn't reload the page, it just puts you the anchor element. And also this won't require further npm dependencies.

Reply:
Yeah, tried that, but didn't work, here's also issue discussing it and suggesting the library I used: https://github.com/remix-run/react-router/issues/394